### PR TITLE
Update Blaze-Persistence to 1.6.7 and unskip all tests

### PR DESCRIPTION
**Pull request by beikov**

### CODE REVIEW

**Comment by gsmet** [comment]
I force pushed the PR to fix the sync.

**Review by aloubyansky — APPROVED** [review]

**Comment by gsmet** [comment]
Thanks!

File: generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-mail/pom.xml [hunk 1]
```diff
@@ -73,7 +73,7 @@
     <dependency>
       <groupId>com.icegreen</groupId>
       <artifactId>greenmail</artifactId>
-      <version>1.6.7</version>
+      <version>${quarkus-blaze-persistence.version}</version>
```
Comment (gastaldi) [inline generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-mail/pom.xml:76]: Is this change intended? It looks like it's affecting a different artifact

Comment (aloubyansky) [reply generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-mail/pom.xml:76]: yes, that's a known issue. The generator is trying to figure out which property to use for versions in test POMs. Sometimes it maps them to an unrelated property that happens to have the needed value. It's weird but harmless. I think there are ways it can be improved. I should get to that at some point.

Comment (gastaldi) [reply generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-mail/pom.xml:76]: @aloubyansky thanks for the explanation. That looks concerning because the intended version gets lost when `quarkus-blaze-persistence.version` is bumped to a new release.
